README.md

```diff
@@ -106,6 +113,26 @@ To see what the environment of your containers is going to look like, run:
 docker compose convert
 ```
 
+#### Production Mode Shortcuts
+
+The commands in the Makefile can be prefaced with PROD. If so, the "dev overlay" configuration in `docker-compose.dev.yml` will be ignored.
+Ports from services other than the HTTP proxy (80/443) will not be exposed. Containers will not mount local directories, watch for changes,
+or rebuild themselves. In theory this should be one way to run Polis in a production environment.
+
+You need a `prod.env` file:
+
+`cp example.env prod.env` (and update accordingly).
+
+Then you can run things like:
+
+```sh
+make PROD start
+
+make PROD start-REBUILD
+
+make PROD start-FULL-REBUILD
```

Comment:
This is great, but needs a warning that it will delete the DB! I wonder if it would be possible to code in a check/confirmation when in prod mode "Do you REALLY want to delete the DB!? (y/n)" :thinking:

Reply:
to delete the "prod" DB you'd have to `make PROD start-FULL-REBUILD`
but maybe there could be something less destructive still. I can imagine a shortcut for rebuilding the images but not dumping the volumes.

Reply:
> to delete the "prod" DB you'd have to `make PROD start-FULL-REBUILD`

Uhh... that's what it says though?

Still, I agree; There should probably be a less destructive option between what is now `start-rebuild` and `start-FULL-REBUILD`. Or maybe even just a separate for rebuilding the db?

FWIW, this has bearing on #1563

Reply:
Added an issue for this: https://github.com/compdemocracy/polis/issues/1643